modyn/supervisor/internal/triggers/trigger.py

```diff
@@ -16,7 +16,6 @@ class TriggerContext:
 
 
 class Trigger(ABC):
-
```

Comment:
I understand a couple of changes are unrelated to the content but mostly empty lines that for whatever reason my local black removes and the CI does not remove. Makes my life a lot easier to merge this.